src/app/services/tasksApi.ts

```diff
@@ -18,7 +19,20 @@ export const tasksApi = api.injectEndpoints({
             query: () => '/tasks/mine',
             providesTags: ['Tasks'],
         }),
+        updateCardContent: builder.mutation({
```

Comment:
What do you think will this file would be the right place to write this API?

Reply:
As this component was related to tasks, so I added this API in this file. Should I create a new file?

Reply:
Also, the unit test for the funcitonality already exists. Should I write unit test for this helper function - "updateCardContent"?

Reply:
the file is okay but the naming doesn't clearly indicate what this function does, read what  it is doing here
```
url: `"${TASKS_URL}/${id}"`,
                    method: 'PATCH',
```

Reply:
yes, we are writing tests for this RTK hook also  "updateCardContent" and the unit tests will also change because of the change in the implementation of the API call.